.travis.yml

```diff
@@ -1,35 +1,39 @@
 language: python
-
 python:
-  - "3.6"
-  - "3.7"
-
+- '3.6'
+- '3.7'
 dist: xenial
 sudo: false
-
+addons:
+  ssh_known_hosts: shrimp.octet.services
 services:
 - docker
-
 before_install:
-  - git clone https://github.com/DEX-Company/barge.git
-  - cd barge
-  - git checkout tags/dex-2019-02-01
-  - bash -x start_ocean.sh --no-brizo --no-pleuston --local-spree-node 2>&1 > start_ocean.log &
-  - cd ..
-  - sleep 15
-
+- git clone https://github.com/DEX-Company/barge.git
+- cd barge
+- git checkout tags/dex-2019-02-01
+- bash -x start_ocean.sh --no-brizo --no-pleuston --local-spree-node 2>&1 > start_ocean.log
+  &
+- cd ..
+- sleep 15
```

Comment:
Is this "sleep 15" basically an optimism that after 15 seconds barge should start?
This may be an example of where we could really use a clear "red light, green light" status.

Reply:
No, since the barge is starting in background it takes a very long time for the barge to startup. So the 15 second sleep is just to allow for barge to get going.